modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -116,8 +134,13 @@ export default class Tile3DLayer extends CompositeLayer {
   }
 
   _onTileLoad(tileHeader) {
+    const {tileset3d} = this.state;
     this.props.onTileLoad(tileHeader);
-    this._updateTileset(this.state.tileset3d);
+    this._updateTileset(tileset3d);
+    if (tileset3d.isLoaded) {
+      this.state.lastUpdatedViewports = this.state.activeViewports;
```

Comment:
This line should be inside `_updateTileset` otherwise `viewportChanged` will repeatedly be set to `true`

Reply:
> > Seems to me that tileset3d.isLoaded() should return false if update is never called.
> 
> It doesn't work
> 
> Update: I found work around erasing `activeViewports` in other place.

Reply:
Done